app/forms/fields/year_date_field.py

```diff
@@ -26,11 +28,28 @@ def data(self):
 
 
 class YearDateField(FormField):
-    def __init__(self, validators, **kwargs):
+    def __init__(
+        self,
+        *,
+        validators: Sequence[Callable],
+        label: str | None = None,
+        description: str,
```

Comment:
All the tests with this have it as an empty string, it could be `str | None = None` same as label since the library defaults it but im wondering if this was for a specific reason and if so all good